src/Screens/LoginScreen.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { useState } from 'react';
+import { View, Text, TextInput, ScrollView, StyleSheet, TouchableOpacity, KeyboardAvoidingView, Platform, Image, SafeAreaView } from "react-native";
+import auth from '@react-native-firebase/auth';
+import Colors from '../Contants/Colors';
+
+
+const LoginScreen = (props) => {
+  const { navigation } = props;
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [emailError, setEmailError] = useState('');
+  const [passwordError, setPasswordError] = useState('');
+  const [credientalError, setCredientalError] = useState('');
+
+  const validateEmail = (email: string) => {
+    if (!email) {
+      setEmailError("Enter Your Email");
+      return false;
+    }
+    const validEmailPattern = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/;

```

Comment:
Please move the validEmailPattern in utils  and make the  function for validate the email so we can reuse it.